setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='express-py',
-    version='2020.10.19.3',
+    version='2021.03.15-rc2',
```

Comment:
I think i mentioned pep440  earlier, but i'll repeat it here https://www.python.org/dev/peps/pep-0440/#pre-releases
Pip get's grumpy if you don't follow the spec for version strings, and falls back to ignoring the semantic meaning and instead just treating them as arbitrary strings. This makes ordering comparisons fail, if you want to do things like ">=2021".

Reply:
Version string has been fixed